src/subscription_manager/injectioninit.py

```diff
@@ -44,7 +43,7 @@ def init_dep_injection():
     # Set up consumer identity as a singleton so we don't constantly re-load
     # it from disk. Call reload when anything changes and all references will be
     # updated.
-    inj.provide(inj.IDENTITY, Identity, singleton=True)
+    inj.provide(inj.IDENTITY, None)
```

Comment:
It is IMHO possible to remove this line. I would also remove `IDENTITY` from `src/subscription_manager/injection.py`.